src/modules/display.js

```diff
@@ -73,6 +76,10 @@ class Todo {
       deleteBtn.addEventListener('click', this.deleteItem);
       const editBtn = element.querySelector('.edit-btn');
       editBtn.addEventListener('click', this.editItem);
+      const checkBtn = element.querySelector('.check-btn');
+      checkBtn.addEventListener('change', (e) => {
+        checkItem(e, Todo);
+      });
```

Comment:
- [ ] Great job The task object's values for `completed` is updated upon user actions. 👏🏻  however the updated array of items must be stored in local storage, so the user gets the correct list values after the page reloads. Actually when the page reloads all the checkboxes get unchecked even though the local storage still shows that their status is true (checked), please fix this and make sure when the page loads the checked box peek is checked.


![image](https://github.com/Othniel2471/todolist-mv/assets/79058364/f26e688c-db7d-41aa-b094-57e3f0d5ad25)